openrouteservice/src/main/java/org/heigit/ors/fastisochrones/FastIsochroneAlgorithm.java

```diff
@@ -184,6 +184,7 @@ public void approximateActiveCells(double approximation) {
             Map.Entry<Integer, IntObjectMap<SPTEntry>> activeCell = activeCellIterator.next();
             if (activeCell.getValue().size() / (double) cellStorage.getNodesOfCell(activeCell.getKey()).size() > approximation) {
                 activeCellIterator.remove();
+                activeCells.remove(activeCell.getKey());
```

Comment:
A few lines above there is an unused local var, and you have an unused import and unused private method. Is your sonar working?

Reply:
Removed